querybook/webapp/components/Survey/Survey.tsx

```diff
@@ -0,0 +1,182 @@
+import clsx from 'clsx';
+import React from 'react';
+import toast, { Toast } from 'react-hot-toast';
+
+import {
+    ISurvey,
+    IUpdateSurveyFormData,
+    SurveySurfaceType,
+    SurveyTypeToQuestion,
+} from 'const/survey';
+import { saveSurveyRespondRecord } from 'lib/survey/triggerLogic';
+import { SurveyResource } from 'resource/survey';
+import { TextButton } from 'ui/Button/Button';
+import { ResizableTextArea } from 'ui/ResizableTextArea/ResizableTextArea';
+
+import { StarRating } from './StarRating';
+
+import './Survey.scss';
+
+export interface ISurveyProps {
+    surface: SurveySurfaceType;
+    surfaceMeta?: Record<string, any>;
```

Comment:
as it appears serval times, define a type for surfaceMeta as `Record<string, any>`?

Reply:
I think its simple enough, I just want to say surface meta is a json serializable dictionary